### PR TITLE
Fixes #1938 Horizontal scaling of web view

### LINKED ISSUE #1938: Allowing user to switch to mobile view
Most of the downloaded content libraries require the user to scroll right to see complete content. Check the screenshot below for reference.
Screenshot_20200321-195237_Kiwix
This hampers the user experience. There must be an OPTIONAL feature to make this display mobile-friendly.

**Describe the solution you'd like**

Add a feature to make the display mobile-friendly.

### PR DESCRIPTION
**Pull request by sonusourav**
Fixes #1938 

### Changes
The function setInitialscale() takes the parameter in percentage. The default value was set to 250, which was causing the trouble. Changed it to 100 and the issue fixed.

### CODE REVIEW

**Comment by sonusourav** [comment]
@macgills  Please review

**Comment by macgills** [comment]
I am awaiting feedback on the issue.

**Comment by sonusourav** [comment]
> 
> 
> I am awaiting feedback on the issue.

@macgills  Feedback from?

**Comment by macgills** [comment]
@sonusourav kelson just commented. Get rid of the zoom preference feature as part of this and use appropriate wideviewport settings for our webviews

**Comment by macgills** [comment]
One instrumented test still failing

**Comment by sonusourav** [comment]
> 
> 
> One instrumented test still failing

@macgills Can you please point exactly which test is failing because when I ran all 82 testes are passing successfully.

image

**Comment by macgills** [comment]
@sonusourav then you must be running the wrong tests
image


```
    onData(allOf(
      is(instanceOf(Preference.class)),
      withKey("pref_zoom_enabled")))
      .perform(click());
```

this is in `testToggle`

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java [hunk 1]
```diff
@@ -82,6 +77,10 @@ public KiwixWebView(Context context, WebViewCallback callback, AttributeSet attr
     settings.setUserAgentString(LanguageUtils.getCurrentLocale(context).toString());
     settings.setDomStorageEnabled(true);
     settings.setJavaScriptEnabled(true);
+    settings.setLoadWithOverviewMode(true);
+    settings.setUseWideViewPort(true);
+    setInitialScale(100);
```
Comment (macgills) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java:82]: I don't think we need to set this

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java [hunk 2]
```diff
@@ -82,6 +77,10 @@ public KiwixWebView(Context context, WebViewCallback callback, AttributeSet attr
...
+    settings.setBuiltInZoomControls(true);
```
[the remark this answers is not in the document]
Comment (sonusourav) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java:83]: `setDisplayZoomControls` was not set to false. Have set it now.

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java (hunk 1 above)
Comment (macgills) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java:82]: I deleted `setInitialScale` and saw no difference, I am using gentoo wiki as the zim as it is the one cited in the ticket. What are you using?

Comment (sonusourav) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java:82]: Try Vikidia

Comment (macgills) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java:82]: Cool, good job

**Pull request merged**